src/main/java/com/facebook/openwifirrm/ucentral/models/AggregatedState.java

```diff
@@ -201,16 +196,28 @@ public boolean equals(Object obj) {
 			Objects.equals(radio, other.radio);
 	}
 
+	/**
+	 * Check whether some other AggregatedState and this one are matached for aggregation.
+	 * If the two match in bssid, station and radio. Then they could be aggregated to one.
```

Comment:
I think: `radio. Then they could be aggregated to one` -> `radio, then they should be aggregated`